src/main/java/core/basesyntax/interfaces/WriteToFile.java

```
@@ -0,0 +1,7 @@
+package core.basesyntax.interfaces;
+
+import java.util.List;
+
+public interface WriteToFile {
+    void write(String filePath, List<String> report);
```

The project must validate inputs and handle null parameters; the description/checklist warns about invalid inputs and null parameters and asks to rethrow informative RuntimeExceptions rather than ignoring exceptions. Ensure implementations of this interface check parameters (filePath and content/report) for null and throw informative RuntimeException if invalid. The description explicitly lists Null parameters as an example of invalid input to anticipate during testing .